### DISCUSSION TITLE
[BUG]创建新邮件400报错，登录admin500报错

### DISCUSSION THREAD

**Discussion by xanthichi** [dreamhunter2333/cloudflare_temp_email#473] [General]
web页面用户部署
# 创建新邮箱
![image](https://github.com/user-attachments/assets/18470dab-7b40-4ba5-b893-348c4232a9c1)
![image](https://github.com/user-attachments/assets/fad6ebf3-7e2e-4342-9589-db12142ccef4)
# admin登录
![image](https://github.com/user-attachments/assets/9796395a-3a51-4bed-81ed-27ba87e09fb1)

**Comment by xanthichi** [comment]
worker页面
![image](https://github.com/user-attachments/assets/9f8e0229-03f9-4fc9-8b9e-bcf16abe94dd)

**Comment by dreamhunter2333** [comment]
你好，是否执行初始化 sql了呢

**Comment by dreamhunter2333** [comment]
需要执行初始化 sql
https://temp-mail-docs.awsl.uk/zh/guide/ui/d1.html

**Reply by liangcm123** [reply]
你好 我也出现了400 创建新邮件失败问题，但是是运行了蛮长一段时间出现的

**Reply by dreamhunter2333** [reply]
> 你好 我也出现了400 创建新邮件失败问题，但是是运行了蛮长一段时间出现的

发送邮件失败么，还是 D1 数据库满了呢

**Comment by liangcm123** [comment]
D1数据库满了，我删除邮件恢复了

Dream Hunter ***@***.***> 于2025年5月20日周二 17:57写道：
